src/index.js

```diff
@@ -0,0 +1,51 @@
+import './scss/app.scss';
+import { content } from './modules/app';
+
+const mainContent = document.getElementById('content');
+
+mainContent.appendChild(content);
+
+const navMenu = document.getElementById('nav_menu');
+
+// container for display each tab content.
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history, your code is clear enough to understand 🙂 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Kindly delete all comments in this file**